BusinessLogicLayer/Initializer/Implementation/Initialize.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using BusinessLogicLayer.Initializer.Interfaces;
+using BusinessLogicLayer.ViewModels;
+using DataAccessLayer.DataBaseModels;
+using Microsoft.EntityFrameworkCore.Metadata;
+using OfficeOpenXml;
+
+namespace BusinessLogicLayer.Initializer.Implementation
+{
+    public class Initialize : IIInitialize
+    {
+        private UnitOfWork _unitOfWork = new UnitOfWork();
+
+        public string CreatePath(string directoryName = "", string fileName = "Languages")
```

Comment:
1) You do not check if file under generated path already exists
2) Please rename method - it does not create, any folder or file. Use "generate" or something else.